hi-ml-cpath/src/health_cpath/utils/output_utils.py

```diff
@@ -282,19 +285,22 @@ def __init__(self, outputs_root: Path, n_classes: int, tile_size: int, loading_p
                                             maximise=maximise)
         self.save_intermediate_outputs = save_intermediate_outputs
         self.tiles_selector: Optional[TilesSelector] = None
+        self.stratify_plots_by = stratify_plots_by
```

Comment:
no need to set this as class attribute, simply pass it to the plots handler similar to tile_size, val_plot_options, ...

Reply:
done, thanks!